R/constants.R

```diff
@@ -280,6 +280,9 @@ CONV_M2_ACR <- 0.0002471058
 CONV_HA_M2 <- 1e4 # ha to m2
 CONV_BM2_M2 <- 1e9
 
+# Money
+CONV_2005_1975_USD = 0.3362  # 2005 USD to 1975 USD
```

Comment:
No - please use the `gdp_deflator` function (or something like that) for this purpose